migration/v0.17.x-v0.18.x/remove-velero-backupstoragelocation.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+here="$(dirname "$(readlink -f "${BASH_SOURCE[0]}")")"
+
+set -euo pipefail
+
+: "${CK8S_CONFIG_PATH:?Missing CK8S_CONFIG_PATH}"
+
+OBJECT_STORAGE_TYPE_SC=$(yq r "${CK8S_CONFIG_PATH}/sc-config.yaml" objectStorage.type)
+if [[ ${OBJECT_STORAGE_TYPE_SC} == "s3" ]]; then
+  STORAGE_TYPE_SC="aws"
+elif [[ ${OBJECT_STORAGE_TYPE_SC} == "gcs" ]]; then
+  STORAGE_TYPE_SC="gcs"
+else
+  exit 0
```

Comment:
This one was on me, I guess we would like it to exit with non-zero code.

And it would also be nice to add a message to what went wrong.

Reply:
EDIT: maybe it should return zero, but at lease we should have a message saying like "skipped deletion of backupstoragelocation because the type was set to ${OBJECT_STORAGE_TYPE_SC}"

Or something like that